app/main/views/drafts.py

```diff
@@ -0,0 +1,154 @@
+from flask import jsonify, abort
+
+from sqlalchemy.exc import IntegrityError
+
+from .. import main
+from ... import db
+from ...validation import is_valid_service_id_or_400
+from ...models import Service, DraftService, ArchivedService
+from ...service_utils import validate_and_return_updater_request, \
+    update_and_validate_service, validate_and_return_service_request, \
+    index_service
+
+
+@main.route('/services/<string:service_id>/draft', methods=['PUT'])
+def create_draft_service(service_id):
+    """
+    Create a draft service from an existing service
+    :param service_id:
+    :return:
+    """
+    is_valid_service_id_or_400(service_id)
```

Comment:
Should this method also fail if a draft already exists with the supplied service ID?